src/app/Scenes/SavedSearchAlert/Components/SavedSearchFilterAdditionalGeneIDs.tsx

```diff
@@ -9,12 +9,13 @@ import { useState } from "react"
 import { TouchableOpacity } from "react-native"
 
 export const SavedSearchFilterAdditionalGeneIDs = () => {
-  const [showAll, setShowAll] = useState(false)
-
   const selectedAttributes = useSearchCriteriaAttributes(
     SearchCriteria.additionalGeneIDs
   ) as string[]
 
+  // If the user has selected any values, show all the options on initial render
+  const [showAll, setShowAll] = useState((selectedAttributes || []).length !== 0)
```

Comment:
I think this could be simplified, but it's completely up to you :)

```suggestion
  const [showAll, setShowAll] = useState(selectedAttributes?.length)
```